salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -375,6 +375,54 @@ server {
     }
     {% endif -%}
 
+    # ali-tm-amin@2024-10-30 proxy journal hosted pages to hubspot
+    {% if pillar.journal.community_newsletter_url %}
+    # Proxy /community-newsletter pages to hubspot
+    location /content-alerts/early-career {
```

Comment:
Probably worth combining these into a single if block for hubspot at this point. They're all identical, so there's no real benefit to having them as separate blocks like this (unlike pubpub where they wanted complex routing needs).